docker/wait-for-db.sh

```diff
@@ -25,10 +25,8 @@ timeout time bash -c 'until PGPASSWORD="${POSTGRES_PASSWORD}" psql \
         --port="${port}" \
         --username "${POSTGRES_USER}" \
         --dbname "${POSTGRES_DB}" \
-        --command '"'"'\q'"'"' > /dev/null 2>&1; do 
+        --command '"'"'\q'"'"' > /dev/null 2>&1; do
   echo "Waiting for postgres to become available...";
   sleep 1;
 done;
-echo "Great news! Postgres is up."';
-
-echo "Timeout, the postgres server is taking too long to respond."
+echo "Great news! Postgres is up."' || echo "Timeout, the postgres server is taking too long to respond." ;
```

Comment:
I'm a little confused by this line. What is it supposed to output the first output if the script is successful and the second one if it's unsuccessful? If so, I'm not sure how the outcome of the connection to the database would affect this line.

Reply:
added if else statement to make it more clear.